### PR TITLE
gateway rename and command changes

### PR DESCRIPTION
**Pull request by ajssmith**

### CODE REVIEW

File: cmd/skupper/skupper_test.go [hunk 1]
```diff
@@ -107,30 +107,29 @@ func TestExposeParseArgs(t *testing.T) {
...
-	assert.Assert(t, b([]string{"oneArg", "twoArg", "threeArg", "fourArg"}))
-	assert.Assert(t, b([]string{"oneArg", "twoArg", "threeArg:fourArg"}))
+	assert.Assert(t, b([]string{"oneArg", "twoArg", "threeArg"}))
+	assert.Assert(t, b([]string{"oneArg", "twoArg:threeArg"}))
 
 	//note  illegal vs extra
-	assert.Error(t, b([]string{"oneArg", "twoArg", "threeArg:fourArg", "fiveArg"}), "extra argument: fiveArg")
-	assert.Error(t, b([]string{"oneArg", "twoArg", "threeArg", "fourArg", "fiveArg"}), "illegal argument: fiveArg")
-	assert.Error(t, b([]string{"oneArg", "twoArg", "threeArg", "fourArg", "fiveArg", "sixArg"}), "illegal argument: fiveArg")
+	assert.Error(t, b([]string{"oneArg", "twoArg:threeArt", "fourArg"}), "extra argument: fourArg")
```
Comment (nluaces) [inline cmd/skupper/skupper_test.go:124]: There is a typo in `threeArt` (minor).

**Review by nluaces — APPROVED** [review]

**Review by grs — APPROVED** [review]

File: cmd/skupper/skupper.go [hunk 2]
```diff
@@ -867,212 +867,221 @@ func IsZero(v reflect.Value) bool {
...
 	}
-
+	cmd.Flags().StringVar(&deleteGatewayName, "name", "", "The name of gateway definition to remove")
 	return cmd
 }
 
-func NewCmdDownloadProxy(newClient cobraFunc) *cobra.Command {
+var downloadGatewayName string
+
+func NewCmdDownloadGateway(newClient cobraFunc) *cobra.Command {
```
[the remark this answers is not in the document]
Comment (fgiorgetti) [reply cmd/skupper/skupper.go:927]: It *could* also explain basically how to install it somewhere.

**Review by fgiorgetti — APPROVED** [review]
@ajssmith still working good. I have added a few comments but I believe we can address them as separate and individual issues later.

File: cmd/skupper/skupper_test.go (hunk 1 above)
Comment (ajssmith) [reply cmd/skupper/skupper_test.go:124]: Thanks, will be fixed in the next update

File: cmd/skupper/skupper.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (ajssmith) [reply cmd/skupper/skupper.go:927]: Updated in the next update to show the file that has been generated.

**Pull request merged**